types/custom-functions-runtime/index.d.ts

```diff
@@ -1,9 +1,11 @@
-// Type definitions for non-npm package Custom Functions 1.5
+
+// Type definitions for non-npm package Custom Functions 1.3
 // Project: https://github.com/OfficeDev/office-js
 // Definitions by: OfficeDev <https://github.com/OfficeDev>,
 //                 Adam Krantz <https://github.com/akrantz>,
 //                 Michelle Scharlock <https://github.com/mscharlock>,
-//                 David Chesnut, <https://github.com/davidchesnut>
+//                 David Chesnut, <https://github.com/davidchesnut>,
+//                 Alison McKay, <https://github.com/alison-mk>
```

Comment:
I don't think there should be a comma here.
```suggestion
//                 David Chesnut <https://github.com/davidchesnut>,
//                 Alison McKay <https://github.com/alison-mk>
```